### ISSUE TITLE
RN 0.56: AccessibilityInfo does not exist in Haste module map. HELP cannot debug on android

### ISSUE THREAD

**Issue by heisty** [facebook/react-native#20266]
Loading dependency graph, done.
error: bundling failed: Error: Unable to resolve module `AccessibilityInfo` from `D:\Cacluator\node_modules\react-native\Libraries\react-native\react-native-implementation.js`: Module `AccessibilityInfo` does not exist in the Haste module map

This might be related to https://github.com/facebook/react-native/issues/4968
To resolve try the following:
  1. Clear watchman watches: `watchman watch-del-all`.
  2. Delete the `node_modules` folder: `rm -rf node_modules && npm install`.
  3. Reset Metro Bundler cache: `rm -rf /tmp/metro-bundler-cache-*` or `npm start -- --reset-cache`.  4. Remove haste cache: `rm -rf /tmp/haste-map-react-native-packager-*`.
    at ModuleResolver.resolveDependency (D:\Cacluator\node_modules\metro\src\node-haste\DependencyGraph\ModuleResolution.js:167:1306)
    at ResolutionRequest.resolveDependency (D:\Cacluator\node_modules\metro\src\node-haste\DependencyGraph\ResolutionRequest.js:80:16)
    at DependencyGraph.resolveDependency (D:\Cacluator\node_modules\metro\src\node-haste\DependencyGraph.js:237:485)
    at Object.resolve (D:\Cacluator\node_modules\metro\src\lib\transformHelpers.js:116:25)
    at dependencies.map.result (D:\Cacluator\node_modules\metro\src\DeltaBundler\traverseDependencies.js:298:29)
    at Array.map (<anonymous>)
    at resolveDependencies (D:\Cacluator\node_modules\metro\src\DeltaBundler\traverseDependencies.js:294:16)
    at D:\Cacluator\node_modules\metro\src\DeltaBundler\traverseDependencies.js:159:33
    at Generator.next (<anonymous>)
    at step (D:\Cacluator\node_modules\metro\src\DeltaBundler\traverseDependencies.js:239:307)
 BUNDLE  [android, dev] ./index.js ░░░░░░░░░░░░░░░░ 1.1% (2/19), failed.
